be/src/olap/rowset/segment_v2/binary_dict_page.cpp

```diff
@@ -240,6 +241,16 @@ void BinaryDictPageDecoder::set_dict_decoder(PageDecoder* dict_decoder, StringRe
 
 Status BinaryDictPageDecoder::next_batch(size_t* n, vectorized::MutableColumnPtr &dst) {
     if (_encoding_type == PLAIN_ENCODING) {
+        auto* col_ptr = dst.get();
+        if (dst->is_nullable()) {
+            auto nullable_col = reinterpret_cast<vectorized::ColumnNullable*>(dst.get());
+            col_ptr = nullable_col->get_nested_column_ptr().get();
+        }
```

Comment:
I think it's a good idea to exposing ColumnDictionary to PageDecoder.
You should deal everything inside the ColumnDictionary.
Such as add a Method for ```IColumn``` named ```convertToCommonColumnIfNecessary```, or some other better way.
Of course, you can add a ```todo``` here.

Reply:
ok